thirdparty/vars.sh

```diff
@@ -409,8 +409,14 @@ LLVM_NAME="llvm-project-16.0.6.src.tar.xz"
 LLVM_SOURCE="llvm-project-16.0.6.src"
 LLVM_MD5SUM="dc13938a604f70379d3b38d09031de98"
 
+#clucene
+CLUCENE_DOWNLOAD="https://github.com/dujijun007/clucene/archive/refs/tags/2.4.4/libclucene-2.4.4.zip"
```

Comment:
any customization to the original source code? Is it possible to use its origin sourceforge download link?

Reply:
@kevincai There are some bugfix commits, how about uploading to the thirdparty repo of StarRocks, in order to do some modifying in the future.

Reply:
yes, we can create a repo under starrocks/ with exactly the same commit from upstream, and then apply necessary bugfixes.

Reply:
please point me the base commit you are referring in your own repo, I can help to create one under starrocks org.

Reply:
You can init from [this commit](https://github.com/dujijun007/clucene/commit/7e80d5bdf6bbe4a6321d488dff455e063f652484)

Reply:
I have created https://github.com/StarRocks/clucene and preserves all the commit history from clucene sourceforge git repo. Can you point me which commit from the original upstream, we can branch from the commit and apply incremental fixes.

Reply:
Thanks for your help, can you publish a raw release with the tar file in this repo? Then I can get the right clucene dependency url.

Reply:
I have made a tag pointed to the main latest commit, you can use the link: https://github.com/StarRocks/clucene/archive/refs/tags/starrocks-2023.11.06.tar.gz